### PR TITLE
stress testing complete

### PR DESCRIPTION
**Pull request by tbright7**

### CODE REVIEW

File: client/src/components/AskQuestion.jsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ class AskQuestion extends React.Component {
   }
   handleSubmit(question) {
     event.preventDefault();
+    console.log('from within handleSubmit on AskQuestion', question)
```
Comment (jnweiner) [inline client/src/components/AskQuestion.jsx:93]: probably want to to take these console.logs out before deploying

File: k6.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export let options = {
+  stages: [
+    { duration: '30s', target: 10 },
+    { duration: '30s', target: 20 },
+    { duration: '30s', target: 40 },
+    { duration: '30s', target: 80 },
+    { duration: '30s', target: 160 },
+    { duration: '30s', target: 0 }
+  ]
```
Comment (ecetino) [inline k6.js:12]: Interesting use of stages. I didn't know you could change parameters at each stage.

**Review by ecetino — APPROVED** [review]
Looks like you had to refactor a lot but nice job getting it done!

File: k6.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+};
+
+export default function () {
+  let listingId = Math.floor((Math.random() * 10000000));
```
Comment (jnweiner) [inline k6.js:16]: I believe this could technically produce 0 as a listing id, I think you need to add a + 1 after (Math.random() * 10000000)

Comment (ecetino) [reply k6.js:16]: or use Math.ceil as well

File: server/index.js [hunk 4]
```diff
@@ -11,42 +13,35 @@ app.use(compression());
...
-    if (err) {
-      console.log(err)
-    } else {
-      res.send(data)
-    }
+app.post('/questions', (req, res) => {
+  const values = [req.body.tripId, `'${req.body.username}'`, `'${req.body.profilepic}'`, `'${req.body.date}'`, `'${req.body.location}'`, req.body.numContributions, req.body.numHelpfulVotes,  `'${req.body.question}'`];
+  const query = `insert into questions (tripId, username, profilePic, date, location, numContributions, numHelpfulVotes, question) 
+  VALUES (${values});`
+  db.queryDB(query, (result) => {
```
Comment (jnweiner) [inline server/index.js:31]: I would look into params to insert values into your sql string to make sure you're escaping characters (https://node-postgres.com/features/queries)

**Comment by jnweiner** [comment]
left a few comments, looks good!

**Pull request merged**